library/pnotes.inc

```diff
@@ -414,9 +414,9 @@ function addPnote(
 
     return sqlInsert(
         'INSERT INTO pnotes (date, body, pid, user, groupname, ' .
-        'authorized, activity, title, assigned_to, message_status) VALUES ' .
-        '(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
-        array($datetime, $body, $pid, $user, $_SESSION['authProvider'], $authorized, $activity, $title, $assigned_to, $message_status)
+        'authorized, activity, title, assigned_to, message_status, update_by, update_date) VALUES ' .
+        '(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, "' . date('Y-m-d H:i:s') . '")',
```

Comment:
Just use NOW() in the mysql (https://www.w3schools.com/sql/func_mysql_now.asp):
```
'(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, NOW())',
```
Also, do this for the below sql statements